qiskit/execute_function.py

```diff
@@ -43,7 +43,8 @@ def execute(experiments, backend,
             seed_transpiler=None, optimization_level=None, pass_manager=None,
             qobj_id=None, qobj_header=None, shots=None,  # common run options
             memory=False, max_credits=None, seed_simulator=None,
-            default_qubit_los=None, default_meas_los=None,  # schedule run options
```

Comment:
This is a breaking change. We cannot just remove these entries.

Reply:
The issue here was that `execute` calls them `default_qubit_los/default_meas_los` but `assemble` calls them `qubit_lo_freq/meas_lo_freq`. So I could just update the `assemble` call on line 299, but that seemed inconsistent. Seems like a better idea to consistent arguments. Also if you did include `default_qubit_los/default_meas_los` it would be valid as part of `run_config`. It just wouldn't do anything, as is the case now

Reply:
Updated this